lessons/03-html-lists/index.html

```diff
@@ -0,0 +1,67 @@
+<html>
+    <head>
+        <title>Hogwarts</title>
+     </head>
+     <body>
+         <h1>Hogwarts</h1>
+         <h2>Houses</h2>
+         <ul>
+             <li><strong>Gryffindor</strong></li>
+             <li>Hufflepuff</li>
+             <li>Ravenclaw</li>
+             <li>Slytherin</li>
+             <title>Notable teachers and staff members</title>
+         </ul>
+         <h3>Notable teachers and staff members</h3>
+         <ol>
+             <li>Argus Filch</li>
+               <li>Filius Flitwick</li>
+                <li>Gilderoy Lockhart</li> 
+               <li>Minerva McGonagall</li>  
+                <li>Poppy Pomfrey</li>
+                <li>Quirinus Quirrell</li>
+                <li>Horace Slughorn</li>
+                <li>Pomona Sprout</li>
+                <li>Sybill Trelawney</li>
+         </ol>
+         <ul>
+            <h4><li>Hogwarts ghosts</li></h4> 
+            </ul>
+            <ol>
+                <li>The Bloody Baron</li>
+                <li>The Fat Friar</li>
+                <li>The Grey Lady</li>
+                <li>Nearly Headless Nick</li>
+            </ol>
+            <ul>
+                   <h5><li>Hogwarts founders</li></h5>
+            </ul>
+            <ol>
+                <li>Godric Gryffindor</li>
+                <li>Helga Hufflepuff</li>
+                <li>Rowena Ravenclaw</li>
+                <li>Salazar Slytherin</li>
+            </ol>
```

Comment:
This is close!

We were supposed to use nested lists in this situation.

So instead of the above, we would want something like:

```html
<ul>
  <li>Hogwarts ghosts
    <ol>
      <li>The Bloody Baron</li>
      ...etc
    </ol>
  </li>

  <li>Hogwarts founders
    <ol>
      <li>Godric Gryffindor</li>
      ...etc
    </ol>
  </li>
</ul>
```